src/test/scala/com/fulcrumgenomics/alignment/MatrixTest.scala

```diff
@@ -72,12 +72,12 @@ class MatrixTest extends UnitSpec {
       }
     }
 
-    val table = m.toString
     val expected =
       """
-        |0  1
-        |0  2
-        |0  3
-      """.stripMargin.trim.replaceAll(" +", "\t")
+        |0 0
+        |0 1
+        |0 2
+      """.stripMargin.trim.replaceAll(" ", "\t")
+    m.toString shouldBe expected
```

Comment:
This test was never testing anything but now that it is, I had to change the "expected" test case in order to get it to pass.